File: limiteMEI-angular/src/app/enums/tipo.empresa.enum.ts
export enum TipoEmpresaEnum {
  MEI_GERAL = 'MEI_GERAL',
  MEI_CAMINHONEIRO = 'MEI_CAMINHONEIRO'
}

export const TIPO_EMPRESA_LABELS: Record<TipoEmpresaEnum, string> = {
  [TipoEmpresaEnum.MEI_GERAL]: 'MEI Geral',
  [TipoEmpresaEnum.MEI_CAMINHONEIRO]: 'MEI Caminhoneiro'
};

export const LIMITE_ANUAL_POR_TIPO: Record<TipoEmpresaEnum, number> = {
  [TipoEmpresaEnum.MEI_GERAL]: 81000,
  [TipoEmpresaEnum.MEI_CAMINHONEIRO]: 251600
};

File: limiteMEI-api/src/main/java/com/limiteMEI/limiteMEI/enums/TipoEmpresaEnum.java
package com.limiteMEI.limiteMEI.enums;

import lombok.Getter;

import java.math.BigDecimal;

@Getter
public enum TipoEmpresaEnum {

    MEI_GERAL("MEI Geral", new BigDecimal("81000.00")),
    MEI_CAMINHONEIRO("MEI Caminhoneiro", new BigDecimal("251600.00"));

    private final String nome;
    private final BigDecimal limiteAnual;

    TipoEmpresaEnum(String nome, BigDecimal limiteAnual) {
        this.nome = nome;
        this.limiteAnual = limiteAnual;
    }
}
